docs/integrations/sources/amazon-seller-partner-migrations.md

```diff
@@ -1,5 +1,40 @@
 # Amazon Seller Partner Migration Guide
 
+## Upgrading to 5.0.0
+
+The PK for the stream `GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL` has been removed, as none composition of the fields could be used as a primary key.
+
+After upgrading, users will need to refresh the source schema and reset this stream if it was previously used in deduplication mode. 
+And if you notice missing data before upgrade, resetting may resolve the issue for the stream.
```

Comment:
I don't know what you're trying to say here.

"If you are experiencing missing data prior to the 5.0.0 upgrade, resetting ___ may resolve issues for the *stream name*."